theme/islandora-videojs.tpl.php

```diff
@@ -1,15 +1,23 @@
-<video id="islandora_videojs" class="video-js vjs-default-skin" controls
- preload="auto" width="640" height="264" 
+<video id="islandora_videojs" <?php if (empty($responsive)): ?>width="640" height="360"<?php endif; ?>
```

Comment:
@pautri Can you also change this `if (empty($responsive))` to a `if (!$responsive)` to make use of the new boolean nature of the variable.